content/maxwell3_fdem/sphere_freespace/dipole_response.rst

```diff
@@ -0,0 +1,91 @@
+.. _dipole_response
+
+The Sphere's Dipolar Response
+-----------------------------
+
+**Purpose**: Here, expressions from Wait (1951) presented for predicting the dipole response from a conductive and magnetically permeable sphere in free-space.
+We show how the characteristic response of the sphere is dependent on its excitation factor; where the excitation factor is used to define the induced dipole moment of the sphere.
+
+.. figure:: ./images/figFreeSpaceSphere.png
+        :align: right
+        :figwidth: 40%
+        :name: FreeSpaceSphere
+
+        Problem geometry for a conductive and magnetically permeable sphere in free-space.
```

Comment:
Is the discussion that follows for a general source, or is it specific to a uniform field?